mlperf_logging/compliance_checker/1.0.0/closed_bert.yaml

```diff
@@ -46,4 +46,4 @@
     REQ:   AT_LEAST_ONE
     CHECK:
         - "'epoch_num' in v['metadata']"
-    ATLEAST_ONE_CHECK: "(v['value'] >= 0.712 if s['global_batch_size'] < 8192 else v['value'] >= 0.706) and v['value'] < 1.0"
```

Comment:
We need clarification from the Training submitters WG if the else term is still needed